embedding/embedlite/utils/DirProvider.cpp

```diff
@@ -36,6 +39,30 @@ NS_IMETHODIMP
 DirProvider::GetFile(const char* aKey, bool* aPersist,
                      nsIFile* *aResult)
 {
+  if (!strcmp(aKey, NS_SYSTEM_SEARCH_DIR_KEY)) {
+    nsCOMPtr<nsIFile> file;
+    nsresult rv;
+#if defined(HAVE_USR_LIB64_DIR) && defined(__LP64__)
+    rv = NS_NewNativeLocalFile(nsDependentCString("/usr/lib64/mozembedlite/chrome/embedlite/content/search-plugins"), false, getter_AddRefs(file));
+#else
+    rv = NS_NewNativeLocalFile(nsDependentCString("/usr/lib/mozembedlite/chrome/embedlite/content/search-plugins"), false, getter_AddRefs(file));
```

Comment:
Not very nice to hardwire these paths but this works for time being at least.

Reply:
Is there any way to retrieve or construct such path?